altapay.php

```diff
@@ -1392,12 +1398,25 @@ private function selectOrder($params)
     public function performCapture($paymentID, $params, $captureRemainedAmount = true, $statusCapture = false)
     {
         try {
-            $api = new Altapay\classes\MerchantAPI();
             $productDetails = new OrderDetail();
             $cart = $this->context->cart;
             $orderSummary = $cart->getSummaryDetails();
-            $api->init($this->getAltapayUrl(), $this->getAPIUsername(), $this->getAPIPassword());
-            $paymentDetails = $api->getPaymentDetails($paymentID);
+            $api = new API\PHP\Altapay\Api\Others\Payments(getAuth());
+            $api->setTransaction($paymentID);
+            $paymentDetails = $api->call();
+
+            $reserved = 0;
+            $captured = 0;
+            $refunded = 0;
+
+            if ($paymentDetails) {
```

Comment:
Can this be something else than an array? If not then there is no need to check if ti contains something as `foreach` will do so.